packages/extension-chakra-storefront/src/pages/cart/index.jsx

```diff
@@ -42,22 +42,21 @@ import {REMOVE_CART_ITEM_CONFIRMATION_DIALOG_CONFIG} from '../../pages/cart/part
 
 // Utilities
 import debounce from 'lodash/debounce'
-import {useCurrentBasket} from '../../hooks/use-current-basket'
+import {useCurrentBasket, useCurrentCustomer} from '../../hooks/'
 import {
     useShopperBasketsMutation,
     useShippingMethodsForShipment,
     useProducts,
     useShopperCustomersMutation
 } from '@salesforce/commerce-sdk-react'
-import {useCurrentCustomer} from '../../hooks/use-current-customer'
 import UnavailableProductConfirmationModal from '../../components/unavailable-product-confirmation-modal'
 import {getUpdateBundleChildArray} from '../../utils/product-utils'
 
 const DEBOUNCE_WAIT = 750
 const Cart = () => {
-    const {data: basket, isLoading} = useCurrentBasket()
+    const {data: basket, isPending} = useCurrentBasket()
```

Comment:
react query has changed isLoading to isPending in v5. See [here](https://tanstack.com/query/latest/docs/framework/react/guides/migrating-to-v5#status-loading-has-been-changed-to-status-pending-and-isloading-has-been-changed-to-ispending-and-isinitialloading-has-now-been-renamed-to-isloading)